vue/components/organisms/size/size.vue

```diff
@@ -98,11 +98,17 @@ export const Size = {
         }
     },
     watch: {
-        enabled(enabled) {
+        enabled: function(enabled) {
             !enabled && this.clear();
         },
-        modelError(error) {
+        modelError: function(error) {
             this.enabled = !error;
+        },
+        active: {
+            handler: function(value) {

```

Comment:
this is weird... having a size component interacting this way with the store... you have a synced `active` prop just use it to control the visibility of the size in the summary I believe it should work

Reply:
... and yes I know you have to propagate that prop all the way up... but that's OK

Reply:
On the other side, we're already committing the size directly (like a boss) to the store so it's just another design mistake...